src/components/AvatarWithImagePicker.js

```diff
@@ -94,7 +89,7 @@ const propTypes = {
     /** Whether navigation is focused */
     isFocused: PropTypes.bool.isRequired,
 
-    ...withLocalizePropTypes,
+    ...withNavigationFocusPropTypes,
```

Comment:
`withNavigationFocusPropTypes` doesn't exist
We already have `isFocused` on line 90